docs/gluster-block.8

```diff
@@ -102,20 +103,34 @@ show help message and exit.
 show version info and exit.
 .PP
 
-.SH SUPPORTED JSON-FORMATS
---json | --json-plain | --json-spaced | --json-pretty
+.SH COMMON CLI OPTIONS
+.SS
+[\fBtimeout <seconds>\fR]
+it is the time in seconds that cli can wait for daemon to respond (default: 300)
+.PP
+
+.SS
+[\fB--json*\fR]
+used to request the output result in json format (default: plain text) and supported JSON formats: --json|--json-plain|--json-spaced|--json-pretty
+.PP
 
 .SH EXAMPLES
 .nf
 To create a block device of size 1GiB
 .B # gluster-block create blockVol/sampleBlock ${HOST} 1GiB
 
+To create a block device of size 1GiB with timeout 150 seconds
+.B # gluster-block timeout 150 create blockVol/sampleBlock ${HOST} 1GiB
+
 To create a block device of size 1GiB with auth enable
 .B # gluster-block create blockVol/sampleBlock auth enable ${HOST} 1GiB
 
 To create a block device of size 1GiB, by preallocating storage with zero fill
 .B # gluster-block create blockVol/sampleBlock prealloc full ${HOST} 1GiB
 
+To create a provision block device of size 1GiB
```

Comment:
Should this be:
To create a thinly-provisioned block device of size 1GiB

Reply:
Fixed.
Thanks.